AttributeError: 'TestMaintenanceVisit' object has no attribute 'make_sales_person'

### Information about bug

Getting error for TC_M_012, TC_M_013, TC_M_014, TC_M_015, TC_M_016 

ERROR: test_update_customer_issue_flag_0_with_previous_partial_visit_TC_M_016 (erpnext.maintenance.doctype.maintenance_visit.test_maintenance_visit.TestMaintenanceVisit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/frappe-bench/apps/erpnext/erpnext/maintenance/doctype/maintenance_visit/test_maintenance_visit.py", line 21, in setUp
    self.sales_person = self.make_sales_person("_Test Sales Person")
AttributeError: 'TestMaintenanceVisit' object has no attribute 'make_sales_person'

----------------------------------------------------------------------
Ran 1 test in 0.239s

FAILED (errors=1)

### Module

other

### Version

Assets: v0.0.1 (pre-prod)

ERPNext: v15.59.0 (pre-prod)

ERPNext CRM: v0.0.1 (pre-prod)

ERPNext India: v0.0.1 (pre-prod)

Frappe Framework: v15.65.2 (pre-prod)

India Compliance: v15.17.1 (pre-prod)

Payments: v0.0.1 (version-15)

Projects: v0.0.1 (pre-prod)

Sales Commission: v0.0.1 (pre-prod)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell

```

@pallavi238848 Getting Bellow issue to all the test cases
======================================================================
FAIL: test_update_customer_issue_flag_0_with_previous_partial_visit_TC_M_016 (erpnext.maintenance.doctype.maintenance_visit.test_maintenance_visit.TestMaintenanceVisit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/frappe-bench/apps/erpnext/erpnext/maintenance/doctype/maintenance_visit/test_maintenance_visit.py", line 199, in test_update_customer_issue_flag_0_with_previous_partial_visit_TC_M_016
    self.assertEqual(wc.resolved_by, sales_person.name)
AssertionError: 't2.service_person' != '_Test Sales Person'
- t2.service_person
+ _Test Sales Person


----------------------------------------------------------------------
Ran 1 test in 0.695s

FAILED (failures=1)

--- a/erpnext/maintenance/doctype/maintenance_visit/test_maintenance_visit.py
+++ b/erpnext/maintenance/doctype/maintenance_visit/test_maintenance_visit.py
@@ -18,7 +18,7 @@ class TestMaintenanceVisit(unittest.TestCase):
 		self.customer = create_customer("_Test Customer", currency="INR")
 		self.item_code = create_item("_Test Item", is_stock_item=1)
 		self.company = "_Test Company"
-		self.sales_person = self.make_sales_person("_Test Sales Person")
+		self.sales_person = make_sales_person("_Test Sales Person")
 
 	def tearDown(self):
 		frappe.db.rollback()

